crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -806,8 +806,7 @@ impl<'a> TimelineEventHandler<'a> {
                         // sent it back without a transaction ID. This occurs
                         // very often right now due to a sliding-sync bug:
                         // https://github.com/matrix-org/sliding-sync/issues/3
-                        // TODO: Raise log level once that bug is fixed
-                        trace!("Received remote echo without transaction ID");
+                        error!("Received remote echo without transaction ID");
```

Comment:
I think `warning!` should be high enough. Also the rest of the comment above should probably be removed?

Reply:
Done.